jekyll/_cci2/contexts.md

```diff
@@ -5,23 +5,24 @@ short-title: "Using Contexts"
 description: "Secured, cross-project resources"
 categories: [configuring-jobs]
 order: 41
-hide: true
 ---
 
+Contexts provide a mechanism for securing and sharing environment variables across projects. The environment variables are defined as name/value pairs and are injected at runtime.
 
-**NOTE:** Contexts are a feature under development. This document is being released early to give you a preview and solicit feedback. We encourage your feedback either on [our Discuss forum](https://discuss.circleci.com/t/contexts-feedback/13908) or in a Pull Request for this document.
+## Overview
+Contexts are created on the Settings page of the CircleCI application, in the Organization section. After a context is set in the application it may be configured in the workflows section of the `config.yml` file for a project.
 
-<hr>
+To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. The default context name is `org-default` and the default rule allows access to all projects in the org.
 
-## Contexts Overview
-Contexts provide centrally managed, secured resources for use in jobs. They are currently primarily used for sharing environment variables across projects.
+## Using a Context
 
-_**Coming Soon:** We will be adding new capabilities to Contexts such the ability to route jobs to certain workers, share encrypted files like certificates across jobs, and new kinds of security rules for who can execute jobs in a Context. What capabilities would you like to see? Let us know in [our Discuss forum](https://discuss.circleci.com/t/contexts-feedback/13908)._
+1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears.
```

Comment:
That is coming soon but i dont fully understand your concern here.  for clarity lets just write "Organization Settings" instead of Settings and it's agnostic.